id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java

```diff
@@ -562,4 +551,9 @@ public ResponseWrapper<CredentialIssueResponse> retriggerCredentialRequest(Strin
 		return credentialIssueResponseWrapper;
 	}
 
+	private LocalDateTime getCurrentDateTime() {
+		DateTimeFormatter format = DateTimeFormatter.ofPattern(EnvUtil.getDateTimePattern());

```

Comment:
This does not need the patterns. You can use the below statement.
`DateUtils.getUTCCurrentDateTime()`

Reply:
Updated as suggested.